content/blog/2021/05/2021-05-15-configure-plugins-with-jcasc.adoc

```diff
@@ -0,0 +1,114 @@
+---
+:layout: post
+:title: "Configure Plugins with JCasC"
+:tags:
+- jenkins
+- jcasc
+- tutorial
+- plugins
+- yaml
+:author: dheerajodha
+opengraph:
+  image: image:/images/post-images/2021-05-15-configure-plugins-with-jcasc/opengraph-image.jpg
```

Comment:
This image is also huge. For opengraph I highly recommend PNG also

Reply:
Done, thanks